components/authz-service/storage/postgres/migration/legacy/v2postgres.go

```diff
@@ -0,0 +1,296 @@
+package legacy
```

Comment:
This logic is ported from postgres

Reply:
This is not our current implementation.This is essentially migration code that won't change. Please see my other comment about how this legacy code will be used :)

Reply:
@bcmdarroch they are copied to this place to be "preserved in time" given that our schema/db calls may change in the future. So the "current" storage implementations might change-- these shouldn't